.github/workflows/nixos-options.yml

```diff
@@ -0,0 +1,30 @@
+name: "Evaluate NixOS options"
+
+permissions:
+  contents: read
+
+on:
+  pull_request_target:
+    paths:
+      - 'nixos/**'
+
+jobs:
+  nixos:
+    name: nixos-options
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          # pull_request_target checks out the base branch by default
+          ref: refs/pull/${{ github.event.pull_request.number }}/merge
+      - uses: cachix/install-nix-action@08dcb3a5e62fa31e2da3d490afc4176ef55ecd72 # v30
+        with:
+          # explicitly enable sandbox
+          extra_nix_config: sandbox = true
+      - name: Building NixOS options
+        run: |
+           nix-build ./nixos/release.nix \
+             -I ./. \
+             --option restrict-eval true \
+             --option allow-import-from-derivation false \
+             -A options
```

Comment:
> NixOS manual's JSON representation as a proxy

Isn't that practically the same as building the nixos manual? We already have a gh action for it I believe.

Reply:
Yes, it is.

Reply:
@JohnRTitor there is one difference I believe. We could have optionsJSON also in staging because it doesn't require stdenv, whereas the manual might causes long rebuilds.

Reply:
Ok, but can we also build the aarch64-linux ones?

Reply:
Unsure if this is possible on x86_64. Does this make a big difference?